src/coreclr/tools/aot/ILCompiler/Program.cs

```diff
@@ -217,7 +217,7 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 syntax.DefineOption("targetarch", ref _targetArchitectureStr, "Target architecture for cross compilation");
                 syntax.DefineOption("targetos", ref _targetOSStr, "Target OS for cross compilation");
 
-                syntax.DefineOption("singlemethodtypename", ref _singleMethodTypeName, "Single method compilation: name of the owning type");
+                syntax.DefineOption("singlemethodtypename", ref _singleMethodTypeName, "Single method compilation: full name of the owning type including assembly name. (Format: NameSpace.TypeName,AssemblyName)");
```

Comment:
The next thing somebody will ask is how to specify generics, nested types, or the parametrized types. Would it be better to just say that it is assembly qualified name or that it is same format as `Type.GetType` ?

Reply:
Something like that `Single method compilation: assembly qualified name of the owning type.`, also probably I have to find place in the documentation where I can put examples of usage. Does this qualifies for couple lines in docs?

Reply:
Yes, sounds reasonable.

Nit: It should be `assembly-qualified name`. The official documentation like https://docs.microsoft.com/en-us/dotnet/api/system.type.gettype?view=net-5.0#System_Type_GetType_System_String_ uses spells it with a dash.

For documentation, I think it would be best to piggy back on https://github.com/dotnet/runtime/blob/main/docs/workflow/debugging/coreclr/debugging-crossgen2.md debugging instructions and improve them as necessary. Once/if we more NativeAOT out of the experimental state, I expect that we will merge crossgen2 and ilc into one compiler that supports multiple compilation modes.

Reply:
Given that this documentation is already in NativeAOT branch, I do not see easy way how better educate developer on the need to read that specific part.